### PR TITLE
`token-db` UI prototype

### PR DESCRIPTION
**Pull request by imxeno**
Resolves L2B-7582, L2B-7678.

### CODE REVIEW

**Review by sdlyy — APPROVED** [review]
mostly nitpicks

File: packages/database/src/token-db/network-rpc/repository.ts [hunk 1]
```diff
@@ -7,8 +7,17 @@ export class NetworkRpcRepository extends BaseRepository {
 
     const rows = records.map(upsertableToRow)
     await this.batch(rows, 100, async (batch) => {
+      console.log(this.db.insertInto('NetworkRpc').values(batch).compile())
```
Comment (sdlyy) [inline packages/database/src/token-db/network-rpc/repository.ts:10]: clog

File: packages/cms/src/server/api/routers/tokens.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { type TokenBridgeRecord, type TokenMetaRecord } from '@l2beat/database'
+import { z } from 'zod'
+import { db } from '~/db'
+import { procedure, router } from '../trpc'
+
+export const tokensRouter = router({
+  tokensFlowDiagram: procedure.THIS_SHOULD_BE_PRIVATE_BUT_IS_NOT_YET.input(
```
[the remark this answers is not in the document]
Comment (imxeno) [reply packages/cms/src/server/api/routers/tokens.ts:7]: it's auth-related, it should check cookies but there's no auth yet

File: packages/cms/src/app/(auth)/tokens/[id]/_components/edit-token-page.tsx [hunk 3]
```diff
@@ -0,0 +1,708 @@
...
+  TableHeader,
+  TableRow,
+} from '~/components/ui/table'
+import { deleteToken, insertToken, updateToken } from '../_actions'
+import '@xyflow/react/dist/style.css'
+import { api } from '~/trpc/react'
+
+const tokenFormSchema = z.object({
+  networkId: z.string().length(21),
+  address: z.string().length(42),
```
[the remark this answers is not in the document]
Comment (imxeno) [reply packages/cms/src/app/(auth)/tokens/[id]/_components/edit-token-page.tsx:66]: not yet

**Pull request merged**